<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>clasament</class>
 <widget class="QDialog" name="clasament">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>491</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>15</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>Clasament general</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>20</y>
     <width>221</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>Clasament per cluburi sportive</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>20</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>Clasament per categorii de lupta</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b_afisezaclasament">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>131</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Afiseaza</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget_2">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>110</y>
     <width>211</width>
     <height>351</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Nume</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Punctaj</string>
    </property>
   </column>
  </widget>
  <widget class="QTableWidget" name="tableWidget_3">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>110</y>
     <width>211</width>
     <height>351</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Nume</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Punctaj</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="b_afisezaclasament_2">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>50</y>
     <width>131</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Afiseaza</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b_afisezaclasament_3">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>50</y>
     <width>131</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Afiseaza</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="clasament_general_pnume">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>110</y>
     <width>91</width>
     <height>351</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QTextEdit" name="clasament_general_punctaj">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>110</y>
     <width>31</width>
     <height>351</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QTextEdit" name="clasament_general_nume">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>91</width>
     <height>351</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>90</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Nume</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>90</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Prenume</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>90</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>P</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="nr_crt">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>21</width>
     <height>351</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="b_inapoi">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>470</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Inapoi</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
